docs/setup-gradle.md

```diff
@@ -726,6 +726,20 @@ Here's a minimal example:
 
 This configuration will automatically apply `v3.17.5` of the [Develocity Gradle plugin](https://docs.gradle.com/develocity/gradle-plugin/), and publish build scans to https://develocity.your-server.com.
 
+Alternatively, you can use the predefined input actions instead of specifying the environment parameters:
```

Comment:
Please make this the "main" way to configure.
The mechanism using environment variables can be demonstrated later, with the table of env vars.

Reply:
main configuration reflects the usage of inputs:
https://github.com/gradle/actions/pull/244/commits/eec8c4dc24f3cad4ad887ab4c76607ec88039941